src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    public static void doWork() {
+        System.out.println("Excavator started its work ");;
+
+    }
+
+    public static void stopWork() {
+        System.out.println("Excavator stopped working ");;
```

Comment:
There is an extra semicolon at the end of the `System.out.println` statement. Please remove it.